src/report/display.rs

```diff
@@ -16,6 +16,7 @@ impl ResultName for FailureReason {
             FailureReason::OOM => "OOM".into(),
             FailureReason::ICE => "ICE".into(),
             FailureReason::NoSpace => "no space left on device".into(),
+            FailureReason::BusErrorWhileLinking => "bus error while linking".into(),
```

Comment:
Can we instead categorize this failure as `Failure::NoSpace`? "bus error while linking" is not that helpful if you don't know what causes it.

Reply:
Ah, I was missing that causal relationship myself. In that case it makes sense to sort the bus error while linking under no space if it is the underlying fault. I will make the change when I am back in front of a computer.

Reply:
I have amended the PRs commit accordingly.